### DISCUSSION TITLE
Project not showing when posting time?

### DISCUSSION THREAD

**Discussion by plolock** [kimai/kimai#5162] [Questions & Answers] [answered]
Hello,
We have recently launched Kimai time tracking as a self-hosted timetracking solution.
We have added a couple of customers with projects and have started some work on creating an integration with [Shortcut](https://www.shortcut.com/)

When posting/creating a time log, it seems we cannot see the related projects sometimes but not always.

What can the issue be here?

Thankful for advice,

**Comment by kevinpapst** [comment]
The projects starts on 22 Nov 2024. Its correct that you cannot see it for a booking on 18 Nov 2024

**Reply by plolock** [reply]
Right, such a stupid error on my end hehe. Thanks :)

**Reply by kevinpapst** [reply, marked as answer]
Which is btw: described in the docs: https://www.kimai.org/documentation/project.html#start-date--end-date
